.github/actions/run-time-exceeded-notification/action.yml

```diff
@@ -0,0 +1,35 @@
+name: send a exceeded build time notification
+description: Sends slack notification if build took more than planned
+
+inputs:
+  max-build-time:
+    description: "Maximum acceptable build time in seconds"
+    required: true
+  slack-token:
+    description: The slack bot token
+    required: true
+  slack-channel:
+    description: The slack channel
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Fetch billable time from gh API
+      id: fetch_time
+      env:
+        GH_TOKEN: ${{ github.token }}
+        total_time: ""
```

Comment:
```suggestion
```

you probably want to fail if `gh api` doesn't return something reasonable?

also when you use $() it should always return an empty string worst case - no need to initialize here